### PR TITLE
Dca 802 base selenium actions

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

**Comment by SergeyMoroz0703** [comment]
Will fix all PEP8 warnings before merging JSD branch to dev

File: app/selenium_ui/jsd/pages/customer_selectors.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    login_submit_button = (By.ID, 'js-login-submit')
+
+
+class CustomerPortalsPageLocators:
+    customer_portals_url = UrlManager().customer_portals_url()
+    search_input_field = (By.ID, 'sd-customer-portal-smart-search-input')
+
+
+class TopPanelSelector:
+
```
Comment (mmizin) [inline app/selenium_ui/jsd/pages/customer_selectors.py:34]: Do we need this empty line?

File: app/selenium_ui/jsd/pages/agent_selectors.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+class DashboardLocators:
+
+    dashboard_url = UrlManager().dashboard_url()
+    dashboard_params = UrlManager().dashboard_params
+    dashboard_window = (By.CLASS_NAME, "page-type-dashboard")
+
+
+class LogoutLocators:
+
```
Comment (mmizin) [inline app/selenium_ui/jsd/pages/agent_selectors.py:50]: Do we need this empty line?

File: app/selenium_ui/jsd/pages/agent_selectors.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    avatar_page_next_button = (By.CSS_SELECTOR, "input[value='Next']")
+    explore_current_projects = (By.CSS_SELECTOR, "a[data-step-key='browseprojects']")
+    login_field = (By.ID, 'login-form-username')
+    password_field = (By.ID, 'login-form-password')
+    login_submit_button = (By.ID, 'login-form-submit')
+    system_dashboard = (By.ID, "dashboard")
+
+
+class DashboardLocators:
+
```
Comment (mmizin) [inline app/selenium_ui/jsd/pages/agent_selectors.py:43]: Do we need this empty line?

**Review by mmizin — APPROVED** [review]

**Review by ometelytsia — APPROVED** [review]

**Review by astashys — APPROVED** [review]

**Pull request merged**